start.py

```diff
@@ -44,5 +44,11 @@ def start_jenkins():
 
 
 if __name__ == "__main__":
-    proc = start_jenkins()
-    sys.exit(proc.wait())
+    proc = None
+
+    try:
+        proc = start_jenkins()
```

Comment:
Right. We discussed that on IRC and I implemented this change. New commit is up.